.github/workflows/update_todo_issue.yaml

```diff
@@ -2,13 +2,13 @@ name: Update TODO Issue
 on:
   push:
     branches:
-      - 'main'
+      - "main"
 
 jobs:
   update_issue:
     name: Update the TODO Issue
     runs-on: ubuntu-latest
-    container: python:3.8.15-slim
+    container: python:3.11.2-slim
```

Comment:
Do we want to ckeep this 3.11.2? Or also change it to 3.8.15 just to be consistent